### DISCUSSION TITLE
[Documentation]  Is onnx computational graph static or dynamic?

### DISCUSSION THREAD

**Discussion by mohsenMahmoodzadeh** [microsoft/onnxruntime#13337] [Other Q&A]
### Describe the documentation issue

As mentioned in the onnxruntime documentation:

```
Out of the box, ONNXRuntime applies a series of optimizations to the ONNX graph, combining nodes where possible and factoring out constant values (constant folding). 
```

My question is:

**Is the exported ONNX computational graph a static graph or a dynamic one?** 

### Page / URL

https://onnxruntime.ai/docs/tutorials/accelerate-pytorch/pytorch.html#inference-with-onnxruntime

**Comment by faxu** [comment]
The ONNX graph is static. You can read more about graph optimizations applied by ONNX Runtime [here](https://onnxruntime.ai/docs/performance/graph-optimizations.html)
